features/step_definitions/item.rb

```diff
@@ -1,26 +1,39 @@
 require 'capybara/cucumber'
-require 'capybara/cuprite'
+require 'selenium/webdriver'
```

Comment:
Please don't change the development environment. It makes tests on the continuous integration server to fail.